apps/microsoft-teams/frontend/src/constants/defaultParams.ts

```diff
@@ -1,21 +1,24 @@
-import { AppInstallationParameters } from '@customTypes/configPage';
+import { AppInstallationParameters, SelectedEvents } from '@customTypes/configPage';
+import { AppEventKey } from './configCopy';
 
 const initialParameters: AppInstallationParameters = {
   tenantId: '',
   notifications: [],
 };
 
+const getDefaultSelectedEvents = () => {
+  const selectedEvents = {} as SelectedEvents;
```

Comment:
(minor comment)

i feel like you could steal from yourself above and use a reduce implementation here instead:

```ts
Object
  .values(AppEventKey)
  .reduce(
    (selectedEvents: SelectedEvents, event) => ({ ...selectedEvents, [event]: false}),
    {}
  )
```